apptest.go

```diff
@@ -407,7 +407,7 @@ func (a *AppSetup) createApps(ctx context.Context, apps []App) error {
 		if app.ValuesYAML != "" {
 			userValuesConfigMap = fmt.Sprintf("%s-user-values", app.Name)
 
-			err := a.createUserValuesConfigMap(ctx, userValuesConfigMap, appCRNamespace, app.ValuesYAML)
+			err := a.ensureUserValuesConfigMap(ctx, userValuesConfigMap, appCRNamespace, app.ValuesYAML)
```

Comment:
Currently we only create the configmap once. 

I think we should update it if it exists. For update tests and for re-running tests locally.